### ISSUE TITLE
Unable to create application. Exiting

### ISSUE THREAD

**Issue by pieterdb4477** [xbmc/xbmc#16065]
### Describe the bug
Every time I try to run the latest Kodi version I get the message "ERROR: Unable to create application. Exiting"

Every time I run the latest Kodi on my Windows 10 desktop, I get the message an "ERROR: Unable to create application. Exiting". I don't get to see anything of Kodi, no splash screen, nothing.

These issues started happening after upgrading from a beta or RC Leia version to the current version and now persist after each first run.  So if I (re-)install kodi, it runs flawlessly. If I quit kodi and start it up afterwards it gives me this error. I tried a clean install (using the checkbox in the installer) but this didn't fix my issue.

## Expected Behavior
Here is a clear and concise description of what was expected to happen:
Kodi starts as usual.

## Actual Behavior

Kodi doesn't run and an error message popups displaying "ERROR: Unable to create application. Exiting"

## Possible Fix

The debug log suggest something is wrong with my addons. I tried a clean reinstall but the issue persists.

### To Reproduce

1. Start Kodi for the first time after an installation
2. Kodi runs flawlessy
3. Quit Kodi
4. Start Kodi again

### Debuglog

The debuglog can be found here:
https://pastebin.com/2cNKaCix

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

Content of the addon folder: https://pastebin.com/4bpeSmX7

### Your Environment
Used Operating system:

 - [ ] Android
 - [ ] iOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Raspberry-Pi
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name: Windows 10 Pro v1809
 - Kodi version: 18.2

Thanks in advance, and a big thanks for developing Kodi 👍 

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by DaVukovic** [comment]
18:50:06.731 T:3824  NOTICE: Starting Kodi (18.0-RC4 Git:20181228-14086a7bd3). Platform: Windows NT x86 64-bit

Could you please update to a more recent Kodi version? The version you are using is long outdated.

**Comment by pieterdb4477** [comment]
> 18:50:06.731 T:3824 NOTICE: Starting Kodi (18.0-RC4 Git:20181228-14086a7bd3). Platform: Windows NT x86 64-bit
> 
> Could you please update to a more recent Kodi version? The version you are using is long outdated.

Thanks, your remark pointed me in the right direction.

Apparently, the old version I had installed was a 32 bit version. I tried to update kodi by installing the x64 version. This created a new installation folder under c:\program files (instead of c:\program files (x86). When I checked "run kodi" in the installer, the x64 ran flawlesly. When I tried to start it by using the icon on my taksbar, the old x86 version ran. I suppose the new x64 version made the manifest unreadable.

Thanks a lot!
